service/src/main/kotlin/fi/espoo/evaka/serviceneed/application/ServiceApplication.kt

```diff
@@ -54,3 +54,14 @@ data class ServiceApplication(
     val additionalInfo: String,
     @Nested("decision") val decision: ServiceApplicationDecision?,
 )
+
+data class UndecidedServiceApplicationSummary(
+    val id: ServiceApplicationId,
+    val sentAt: HelsinkiDateTime,
+    val childId: ChildId,
+    val childName: String,
+    val startDate: LocalDate,
+    val placementEndDate: LocalDate,
+    val currentNeed: String?,
+    val newNeed: String?,
```

Comment:
Tähän populoitava tieto näyttäisi olevan non-nullable `service-need-option` taulussa. Voisiko sen vaihtaa tännekkin non-nullableksi?
```suggestion
    val newNeed: String,
```

Reply:
taitaa voida https://github.com/espoon-voltti/evaka/pull/5715/commits/34e1614d05afc02023bc65b0324f41d8646f131c